docs/Install/Installing_HPE_Swarm_Learning_Management_UI.md

```diff
@@ -0,0 +1,27 @@
+# <a name="GUID-60017971-B0A9-4119-AEAF-A21594EE5C1E"/> Installing HPE Swarm Learning Management UI \(SLM-UI\)
+
+Installing Swarm Learning is a two-step process.
+
+1.  Using SLM-UI Installer, you can install the SLM-UI on one host.
+2.  Using SLM-UI, you can install SL in multiple hosts and run the examples.
+
+## Procedure
```

Comment:
This file can be deleted..  This way of pulling images is ap[applicable only for enterprise and not community users.

Reply:
Resolved